p2p/pex/pex_reactor.go

```diff
@@ -260,9 +260,16 @@ func (r *PEXReactor) SetEnsurePeersPeriod(d time.Duration) {
 
 // Ensures that sufficient peers are connected. (continuous)
 func (r *PEXReactor) ensurePeersRoutine() {
-	// Randomize when routine starts
-	ensurePeersPeriodMs := r.ensurePeersPeriod.Nanoseconds() / 1e6
-	time.Sleep(time.Duration(rand.Int63n(ensurePeersPeriodMs)) * time.Millisecond)
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Doesn't this have global effect ? Maybe we should use a localized seed in case this bites us when we're looking for determinism somewhere later

Reply:
see https://github.com/tendermint/tmlibs/blob/develop/common/random.go

Reply:
Good catch.

@melekes Unfortunately `common` doesn't offer a method similar to `Int63n`, is worth adding it and using here?

Gonna go for local seed in the meantime.